src/main/java/io/airlift/airline/ParseOptionConversionException.java

```diff
@@ -33,6 +33,14 @@
         this.typeName = typeName;
     }
 
+    ParseOptionConversionException(String optionTitle, String value, String typeName, Throwable cause)
+    {
```

Comment:
Since the `cause` is nullable in exception classes' constructors, you can have the previous constructor delegate to the new. This would allow to avoid duplication of message formatting.

Reply:
Absolutely correct. Fixed that.